staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1689,6 +1709,14 @@ const (
 	MountPropagationBidirectional MountPropagationMode = "Bidirectional"
 )
 
+// volumeDevice describes a mapping of a raw block device within a container.
+type VolumeDevice struct {
+	// Required: This must match the Name of a Volume in the pod.
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
+	// Required: Path inside of the container that the device will be mapped to.
+	DevicePath string `json:"devicePath" protobuf:"bytes,2,opt,name=devicePath"`
```

Comment:
// devicePath is the path...

Reply:
ack...fixed

Reply:
Needs be `// devicePath is the path inside the container ... A value must be provided.`

Reply:
Same for the one above.

Reply:
Actually, you can drop `A value must be provided` because the fields is already marked as required by default.